src/vms/pvm/txs/fee/utils.ts

```diff
@@ -0,0 +1,94 @@
+import { Bytes } from '../../../../serializable';
+import { ID_LEN } from '../../../../serializable/fxs/common/id';
+import { INT_LEN } from '../../../../serializable/primitives/int';
+import { bufferToNumber } from '../../../../utils';
+
+// TODO: Why is there this offset on the message? What are the first two bytes?
+// Offset the start of deserialization because of some unknown bytes at the front.
+const START_INDEX = 2;
```

Comment:
I have no idea what the first two bytes are in warp messages. It does't align with what I'm seeing in the code or docs. I assume this must be some sort of type or codec id or something?

Future Warp classes with deserialization/serialization will need to account for this.